src/envo/Makefile

```diff
@@ -393,6 +393,7 @@ mirror/bfo.owl: envo-edit.owl
 .PRECIOUS: mirror/%.owl
 
 mirror/uberon.owl: 
+	wget http://purl.obolibrary.org/obo/uberon.obo # suggest fix for parser error; see https://github.com/EnvironmentOntology/envo/pull/1289
```

Comment:
@cmungall added `wget` command to download `uberon.obo`. The qc-check is still failing :(